### ISSUE TITLE
new-line-before-expect doesn't allow multiline expects.

### ISSUE THREAD

**Issue by C0DK** [tlvince/eslint-plugin-jasmine#354]
**Describe the bug**
[new-line-before-expect](https://github.com/tlvince/eslint-plugin-jasmine/blob/master/docs/rules/new-line-before-expect.md) doesn't see preceding expect if it's in a multiple case.

i.e this fails (on the last line):
```typescript
    expect("foo").toEqual("foo");
    expect("foo").toEqual(
      "foo"
    );
    expect("foo").toEqual("foo");
```
This does not 
```typescript
    expect("foo").toEqual("foo");
    expect("foo").toEqual("foo");
    expect("foo").toEqual("foo");
```

**To Reproduce**

> Steps to reproduce the behaviour

```typescript
describe('the bug', () => {
  it('happens', () => {
    expect('foo').toEqual('foo');
    expect('foo').toEqual('foo');
    expect('foo').toEqual('foo');
  });
});
```

**Expected behaviour**

That it wouldn't fail

**Context**

- Node.js version: 17.9.0
- eslint-plugin-jasmine version: 4.1.3
- eslint version:
- Operating system:
